core/server/services/mega/mega.js

```diff
@@ -305,10 +304,7 @@ async function handleUnsubscribeRequest(req) {
     }
 
     try {
-        let memberData = {subscribed: false};
```

Comment:
I feel like this should be erroring - is `handleUnsubscribeRequest` still used in 5.0?